docs/src/technik/development/index.md

```diff
@@ -36,6 +36,25 @@ flowchart LR
     end
 ```
 
+## Services und Ports
+
+Übersicht über die Services und auf welchen Port im Standard lauschen
+
```

Comment:
fixed with 6e8486ad9af0676ffe30b3f7779e7a0ae3edc9a3

I had to remove that last sentence because a documentation should describe the current situation and not possible or unclear states in an unknown future.